src/en/juju-backups.md

```diff
@@ -0,0 +1,405 @@
+Title: Back up and restore Juju
```

Comment:
I added these where they were missing, apart from the help command where it just seemed ridiculous because the only text there is basically a usage.